test/unit/spec/v2/view-builder/internals/BaseFooter_spec.js

```diff
@@ -125,5 +131,30 @@ describe('v2/view-builder/internals/BaseFooter', function () {
     ]);
   });
 
+  it('does not add signout link when features.hideSignOutLinkInMFA is true, even if `showSignoutLink` is true', () => {
```

Comment:
do we need a test case for `features.hideSignOutLinkInMFA` is true?

Reply:
oh sigh... yes, actually that is the one we need and that is what I meant to do... fixing it